Passport.js

```diff
@@ -0,0 +1,58 @@
+import passport from 'passport'
+import passportJWT from 'passport-jwt'
+import { Pool } from 'pg';
+
+import DbQuery from './server/model/';
+import { comparePwd } from './server/helpers/pwdHash';
+
+const JWTStrategy = passportJWT.Strategy;
+const ExtractJWT = passportJWT.ExtractJwt;
+const LocalStrategy = require('passport-local').Strategy;
+
+passport.use(new JWTStrategy({
+	jwtFromRequest: ExtractJWT.fromHeader('authorization'),
+	secretOrKey: 'secret'	
```

Comment:
You might want to take this out from your code and place it in a `.env` file

Reply:
Noted